pom.xml

```diff
@@ -103,24 +101,28 @@
         <commons.fileupload.version>1.3.2</commons.fileupload.version>
         <logback.classic.version>1.1.7</logback.classic.version>
         <usng4j.version>0.1</usng4j.version>
-        <!--camel dependencies-->
-        <camel.version>2.18.3</camel.version>
-        <servicemix-specs.version>2.7.0</servicemix-specs.version>
+        <!--START versions for camel features in platform-app-->
+        <!--DO NOT EDIT versions unless updating camel-->
+        <camel.version>2.19.0</camel.version>
         <geronimo-jta-spec.version>1.1.1</geronimo-jta-spec.version>
         <geronimo-jms-spec.version>1.1.1</geronimo-jms-spec.version>
-        <qpid-jms-client.version>0.11.0</qpid-jms-client.version>
+        <qpid-jms-client.version>0.11.1</qpid-jms-client.version>
         <qpid-proton-j.version>0.14.0</qpid-proton-j.version>
-        <camel.netty.version>4.1.5.Final</camel.netty.version>
+        <camel.netty.version>4.0.45.Final</camel.netty.version>
```

Comment:
I'm sure you copied it from there stuff but it is a little weird that the version went down here.